lib/main.coffee

```diff
@@ -17,11 +17,12 @@ module.exports =
       type: 'integer'
       default: 100
       order: 2
-    maxSuggestions:
-      title: 'Maximum Suggestions'
-      description: 'The list of suggestions will be limited to this number.'
+    maxVisibleSuggestions:
+      title: 'Maximum Visible Suggestions'
+      description: 'The autocomplete popup will only show this many suggestions.'
```

Comment:
We've been calling it the "suggestion list". Should be consistent one way or the other.